### PR TITLE
The hardcoded ProvileHeaderView height replaced by the top constraint for the nameLabel

### PR DESCRIPTION
**Pull request by ZuleykhaPavlichenkova-TomTom**
Screenshot 2024-04-18 at 20 38 29

### CODE REVIEW

**Comment by ZuleykhaPavlichenkova-TomTom** [comment]
Thinking of removing the phone number label

File: ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift
Comment (ZuleykhaPavlichenkova-TomTom) [inline ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift:1]: Screenshot 2024-04-19 at 10 09 51

File: ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift [hunk 1]
```diff
@@ -151,7 +148,7 @@ final class OverviewContactViewController: UIViewController, MFMessageComposeVie
         setUpMainStackView()
         setUpProfileHeaderView()
         setUpButtons()
-        setUpPhoneNumberView()
+//        setUpPhoneNumberView()
```
Comment (pavoleg2009) [inline ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift:151]: please remove commented code

File: ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift [hunk 2]
```diff
@@ -217,6 +212,12 @@ final class OverviewContactViewController: UIViewController, MFMessageComposeVie
     
     private func setUpDeleteButton() {
         mainStackView.addArrangedSubview(deleteButton)
+        NSLayoutConstraint.activate([
+//            deleteButton.topAnchor.constraint(equalTo: buttonsStackView.bottomAnchor, constant: 32),
+//            deleteButton.bottomAnchor.constraint(equalTo: backgroundView.bottomAnchor, constant: -32),
+//
```
Comment (pavoleg2009) [inline ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift]: commented code should not be in the diff

**Review by pavoleg2009 — CHANGES_REQUESTED** [review]

File: ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift (hunk 2 above)
Comment (ZuleykhaPavlichenkova-TomTom) [reply ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift]: Cleaned up

File: ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift (hunk 1 above)
Comment (ZuleykhaPavlichenkova-TomTom) [reply ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift:151]: Done

**Pull request merged**